src/internal/tokenizer_test.ts

```
@@ -0,0 +1,2674 @@
+      const locationTracker = new LocationTracker();
+      const state = {
+        locationTracker,
+        locationSnapshot: locationTracker.snapshot(),
+        type: Mode.Literal,
+        buffer: "foobar",
+      } as LiteralMode;
+      const tokens: Token[] = [];
+      let token!: Token;
+
+      await t.step("flushState should return a token", () => {
+        assertExists(token = flushState(state, tokens)!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)